### PR TITLE
Xenobio remap and quick fix

### LINKED ISSUE #9951: [BUG] Mobs can attack while inside containers
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

23070821

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

When placing a mob such as a xenomorph queen inside a locker, the mob will still attack and harm players next to the locker, even if the locker is closed and welded.

### What should have happened instead?

Mobs should not be able to attack players when they are inside a closed container. 
Should be as simple as adding a check to hostile mob attack logic checking if they are inside a container first.

### Steps to reproduce the issue

1. Spawn a hostile mob with a melee attack such as a space carp or xenomorph queen.
2. Open a locker a lure the creature onto the lockers tile.
3. Close the locker on the mob, trapping it inside.
4. Stand next to the creature trapped inside the locker.
5. The mob will continue attacking you despite being trapped.

### Additional context

Known issue for awhile, just making sure their is an issue for it.

### PR DESCRIPTION
**Pull request by AtnerNT**
Closes #9951 

Capture

### Changelog:
CL: [Improvement] Remaps ministation's xenobiology lab
CL: [Fix] Fixed mobs being able to attack from within lockers and containers

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
I'll wait for @Bod9001 approval because I'm not entirely sure yet how the requirements of xenobio here are met or not

**Comment by Bod9001** [comment]
I would say use usual disposal bins, since you might accidentally walk into them, and make sure the ACU has no access requirement

**Comment by AtnerNT** [comment]
> I would say use usual disposal bins, since you might accidentally walk into them, and make sure the ACU has no access requirement

applied

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**
